secret_manager.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+function main() {
+    gcloud_auth "$INPUT_GCP_SA_KEY" 
```

Comment:
im curious, do we need to pass these into the function or can that just be defined in the function?